Documentation/porting-imx.md

```diff
@@ -410,6 +410,20 @@ Edit `Dsdt-Sdhc.inc` and ensure that the SDHC instance on which your boot media
 ```c
 Package (2) { "Regulator1V8Exist", 0 }, // 1.8V Switching External Circuitry: Not implemented
 ```
+### PWM
+
+For best experience using the PWM WinRT APIs from UWP apps, some device properties may need to be set where some of which are generally required for UWP access to the PWM device interface while others are required for some PWM WinRT APIs to function correctly. More on these device interface properties can be found here in the [Setting device interface section](https://docs.microsoft.com/en-us/windows-hardware/drivers/spb/pulse-width-controller%20driver#setting-device-interface-properties) of the PWM DDI MSDN documentation.
+
+For example on setting the PWM deivce interface properties statically from an inf file, see the [Virtual PWM driver sample here](https://github.com/Microsoft/Windows-iotcore-samples/tree/develop/Drivers).
```

Comment:
```suggestion
For an example of setting the PWM deivce interface properties statically from an inf file, see the [Virtual PWM driver sample](https://github.com/Microsoft/Windows-iotcore-samples/tree/develop/Drivers).
```

Reply:
We shouldn't merge this pull until the virtual PWM driver has been checked into Windows-iotcore-samples/Drivers, and this URL should be updated directly to its folder.